koiki/tests/test_client.py

```diff
@@ -78,8 +89,9 @@ def test_create_delivery_successful_response(self, mock_logger):
     @responses.activate
     @patch('koiki.client.logging', autospec=True)
     def test_create_delivery_failed_response(self, mock_logger):
-        responses.add(responses.POST, 'https://testing_host/rekis/api/altaEnvios',
-                      json={'error': 'Bad Request'}, status=400)
+        responses.add(responses.POST, 'https://testing_host/rekis/api/altaEnvios', status=400,
```

Comment:
In the long term we could change the status codes using rest_framework.status and maybe avoid returning the json error since it does not provide additional programmable information.

Reply:
we could perhaps reconsider what we do with responses, yes. Other than logging them for debugging purposes, no one will read them.